source/libs/api.js

```diff
@@ -1,11 +1,19 @@
+import OptionsSync from 'webext-options-sync';
+
 const cache = new Map();
+const headers = {};
+const options = new OptionsSync().getAll();
 
 export default async endpoint => {
```

Comment:
Add other arguments like `method`, and `data` (in case we want to use other methods like `POST` and send some data)?

Reply:
> in case we want to

They'll be added when they're actually needed

Reply:
v4 API needs `POST`, am I right?

Reply:
No feature currently uses v4

Reply:
Shouldn't they?